### ISSUE TITLE
Where is the improvement come from?

### ISSUE THREAD

**Issue by frostinassiky** [ChenJoya/2dtan#2]
Hi Joya,

Thanks again for sharing your brilliant work! 

From your experience, where is the improvement come from? Why does the same architecture (2D-TAN) give very different performances?

**Comment by ChenJoya** [comment]
Our clean implementation speeds up the training and inference. I think that the improvement on recall@k comes from the learning hyper-parameters (learning rate, milestone). We use larger learning rate and less epoches.

**Comment by frostinassiky** [comment]
Thanks for your response! Again it is very impressive to see the boost!
